backend/src/main/kotlin/app/ehrenamtskarte/backend/auth/webservice/schema/ResetPasswordMutationService.kt

```diff
@@ -35,6 +37,10 @@ class ResetPasswordMutationService {
                 val key = AdministratorsRepository.setNewPasswordResetKey(user)
                 Mailer.sendResetPasswordMail(backendConfig, projectConfig, key, email)
             }
+            if (user == null) {
+                val context = dfe.getContext<GraphQLContext>()
+                logger.info("${context.remoteIp} $email failed to request password reset mail")
+            }
```

Comment:
I think this is not the problematic endpoint.

The problematic one is the `resetPassword` endpoint!